src/cmd/graph_pwsh/README.md

```diff
@@ -82,10 +84,25 @@ docker run --rm -it -v $(pwd):/usr/pwsh --env-file env_names corso/graph_pwsh \
        pwsh -c "Get-MgUserMessage -UserId <userID or UPN> -MessageID <messageID>"
 ```
 
+### Running a remote script
+
+In some cases, it may be prudent to run a more complex Powershell script that strings
+a number of commands together. While it is possible to compact the sctipt to a single line,
+that is quite inconvenient.
+
+Instead, it is possible to stash the desired script somewhere publically accessible (e.g. Gist)
+and execute the container which will pull it and then execute it. This can be done using the
+`Execute-Script.ps1` command as follows.
+
+```sh
+docker run --rm -it -v $(pwd):/usr/pwsh --env-file env_names corso/graph_pwsh \
```

Comment:
Thoughts on adding this to a makefile inside cmd/graph_pwsh?  Most of this looks like boilerplate.  Would be nice to call:
```
make execute-pwsh url=<download URL>
```